src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -567,11 +569,30 @@ def _library_dep_order(library: str):
             case _:
                 return 3
 
-    def _upload_wheel(self):
+    def _upload_wheelhouse(self, path: Path) -> list[str]:
+        """Upload a wheelhouse.
+
+        TODO
+        ----
+        Move this method into the WheelsV2 class.
+        """
+        # We can not upload a .zip file to the workspace as the platform fails when extracting all files inside the zip
```

Comment:
There are some mentions of a wheelhouse on the Databricks documentation pages (see below). BUT:
- if I upload a zip file to the workspace via the Databricks UI, the zip is extracted
- Uploading a zip via the API does not work
- The docs mention the wheelhouse should have the following extension: `.wheelhouse.zip`

The following also works, but it is not using a wheelhouse within Databricks

Doc pages:
- https://docs.databricks.com/en/jobs/jobs-2.0-api.html#library
- https://docs.databricks.com/en/libraries/cluster-libraries.html#install-a-library-on-a-cluster